spec/craft_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rspec'
+require './lib/person'
+require './lib/craft'
+require './lib/event'
+
+RSpec.describe Craft do
+  it 'exists' do
+    craft = Craft.new('knitting', {yarn: 20, scissors: 1, knitting_needles: 2})
+
+    expect(craft).to be_instance_of(Craft)
+  end
+
+  it 'has a name' do
+    craft = Craft.new('knitting', {yarn: 20, scissors: 1, knitting_needles: 2})
+
+    expect(craft.name).to eq("knitting")
+  end
+
+  it 'has supplies required' do
+    craft = Craft.new('knitting', {yarn: 20, scissors: 1, knitting_needles: 2})
+
+    expect(craft.supplies_required).to eq({:yarn=>20, :scissors=>1, :knitting_needles=>2})
+  end
```

Comment:
Per my comment above, with the way your Craft class is written, every Craft has the same supplies. So this test will pass, but if you try to create a second craft with different supplies it will still have `{yarn: 20, scissors: 1, knitting_needles: 2}` as it's supplies_required